compile_commands.json

```diff
@@ -0,0 +1,3092 @@
+[
```

Comment:
This is not something that should be committed here.

Reply:
Nice catch - will remove